content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md

```diff
@@ -0,0 +1,401 @@
+---
+title: "Install and Manage MAAS Gateway"
+metaTitle: "Install and Manage MAAS Private Cloud Gateway"
+metaDescription: "Learn how to install and manage the MAAS Private Cloud Gateway in Palette."
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+
+# Overview 
+
+The Private Cloud Gateway (PCG) enables support for private cloud or data center environments. You can set up the cloud gateway as a single-node or three-node cluster based on your requirements for high availability (HA).  
+<br />
+
+<InfoBox>
```

Comment:
See the comment in the prerequisite section for how we can remove this callout box.

Reply:
Added to one of the prereqs